extension/extension_test.go

```diff
@@ -38,15 +61,15 @@ func TestFilterForUpdates(t *testing.T) {
 	testExtensionsMap.StoreExtensions(&testExtensions)
 
 	// No updates when nothing to check
-	updateRequest := UpdateRequest{}
-	check := updateRequest.FilterForUpdates(testExtensionsMap)
+	emptyExtensions := Extensions{}
+	check := filterForUpdates(emptyExtensions, testExtensionsMap)
```

Comment:
What's the reason for calling the version above? Instead of the actual one (ex: `emptyExtensions.FilterForUpdates(testExtensionsMap)`)